### ISSUE TITLE
 Exception: 'NoneType' object has no attribute 'unpack'

### ISSUE THREAD

**Issue by shakaran** [nE0sIghT/chrome-gnome-shell-mirror#37]
I get the following trace in Ubuntu 17.04 development branch

```
[20061] Uncaught exception of type <class 'AttributeError'> occured
  File "/usr/bin/chrome-gnome-shell", line 226, in on_input
    self.process_request(request)
  File "/usr/bin/chrome-gnome-shell", line 375, in process_request
    'shellVersion': shell_version.unpack(),
[20061] Exception: 'NoneType' object has no attribute 'unpack'

```

**Comment by nE0sIghT** [comment]
Do you using Unity? Get me screenshot of your desktop

**Comment by shakaran** [comment]
I am using Ubuntu 17.04 with Gnome Session (Metacity). Sorry I cannot put a public screenshot here, I have confidential data

**Comment by nE0sIghT** [comment]
Show output of ```ps -ef | grep 'unity\|compiz\|gnome-shell'```

**Comment by shakaran** [comment]
```
$ ps -ef | grep 'unity\|compiz\|gnome-shell'
root      2776     1  0 mar04 ?        00:00:00 /usr/lib/x86_64-linux-gnu/unity-greeter-session-broadcast/unity-greeter-session-broadcast-service
shakaran  8495  2853  0 mar04 ?        00:00:00 /usr/lib/gnome-shell/gnome-shell-calendar-server
shakaran 20060 19648  0 02:07 pts/8    00:00:00 /usr/bin/python3 /usr/bin/chrome-gnome-shell chrome-extension://gphhapmejobijbbhgpjhcjognlahblep/
shakaran 20112 10660  0 20:03 pts/6    00:00:00 grep --color=auto unity\|compiz\|gnome-shell

```

**Comment by nE0sIghT** [comment]
As you can see there is no GNOME Shell running. You can not use GNOME Shell integration and GNOME Shell extensions without GNOME Shell.

GNOME Shell looks like this:
![image](https://cloud.githubusercontent.com/assets/3389586/23590396/8953827c-01f8-11e7-8eb8-475e193f4795.png)

**Comment by nE0sIghT** [comment]
Probably, you are using GNOME Flashback:

![image](https://cloud.githubusercontent.com/assets/3389586/23590425/1ec35dbe-01f9-11e7-8d7d-2f73d3be388e.png)

**Comment by shakaran** [comment]
ok, could you detect in some way the enviroment variable or check at least the exception for avoid the traceback in the terminal?

**Comment by nE0sIghT** [comment]
It's done already in #35 and available in 8.2 release of connector

**Comment by shakaran** [comment]
@nE0sIghT ok, thanks
